packages/flutter_tools/lib/src/commands/build_ios_framework.dart

```diff
@@ -73,16 +73,17 @@ class BuildIOSFrameworkCommand extends BuildSubCommand {
               'By default, all build configurations are built.'
       )
       ..addFlag('universal',
-        help: 'Produce universal frameworks that include all valid architectures. '
-              'This is true by default.',
-        defaultsTo: true,
-        negatable: true
+        help: '(Deprecated) Produce universal frameworks that include all valid architectures. '
+              'This option will be removed in a future version of Flutter.',
+        negatable: true,
+        hide: true,
       )
       ..addFlag('xcframework',
-        help: 'Produce xcframeworks that include all valid architectures (Xcode 11 or later).',
+        help: 'Produce xcframeworks that include all valid architectures.',
+        defaultsTo: true,
       )
       ..addFlag('cocoapods',
-        help: 'Produce a Flutter.podspec instead of an engine Flutter.framework (recommended if host app uses CocoaPods).',
+        help: 'Produce a Flutter.podspec instead of an engine Flutter.xcframework (recommended if host app uses CocoaPods).',
       )
       ..addOption('output',
         abbr: 'o',
```

Comment:
it seems like we never built in flag options to let users choose the target-platforms (like Android). Do you think we should?

Reply:
It's `flutter build ios-framework` it doesn't support other platforms.